src/API/Google/ApiExceptionTrait.php

```diff
@@ -44,6 +44,37 @@ protected function has_api_exception_error( ApiException $exception, string $err
 		return false;
 	}
 
+	/**
+	 * Returns a list of detailed errors from an ApiException.
+	 * If no errors are found the default Exception message is returned.
+	 *
+	 * @param ApiException $exception Exception to check.
+	 *
+	 * @return array
+	 */
+	protected function get_api_exception_errors( ApiException $exception ): array {
+		$errors = [];
+		$meta   = $exception->getMetadata();
+
+		if ( is_array( $meta ) ) {
+			foreach ( $meta as $data ) {
+				if ( ! is_array( $data['errors'] ) ) {
+					continue;
+				}
+
+				foreach ( $data['errors'] as $error ) {
+					if ( ! empty( $error['message'] ) ) {
+						$error_code            = reset( $error['errorCode'] ) ?: 'ERROR';
```

Comment:
💅 

If `$error['errorCode'] == '0'` then `$error_code` will be 'ERROR'  instead of '0'

WDTY of
`$error_code            = $error['errorCode'][0]  ?? 'ERROR';`